### DISCUSSION TITLE
My Webhooks are triggering multiple times (like if there was a cron,except the data is the same)

### DISCUSSION THREAD

**Discussion by fbatiga** [supabase/supabase#15625] [Questions]
Hello,

Has any one ever experienced an issue where webhooks (on INSERT) are triggered multiple times et repeated over many days. The hook content is the same...

I've tried deleting the hook and creating another one, the issue is still present :/

yelp !

**Comment by richmeij** [comment]
I have witnessed the same behavior. Ive also noticed that the net.http_request_queue table doesnt get cleaned. Im thinking this might be the cause, but Im not sure why this cleanup doesnt happen.

**Comment by fbatiga** [comment]
In my case the problem was resolved by disabling the cron and http
extension, wiping the related tables and reenabling them...

**Reply by richmeij** [reply]
Thank you! In my case it was the pg_net extension, but that did the trick!

**Comment by richmeij** [comment]
> In my case the problem was resolved by disabling the cron and http extension, wiping the related tables and reenabling them...

Interesting, thanks, I will try that. Upon examination I dont see the cron nor http extension enabled in my DB extensions tab. But, I do have the pg_net extension. I will try to disable/enable that.
